config/environments/development.rb

```diff
@@ -9,16 +9,27 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
-  # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  # Show full error reports.
+  config.consider_all_requests_local = true
+
+  # Enable/disable caching. By default caching is disabled.
+  # Run rails dev:cache to toggle caching.
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.perform_caching = false
+
   # Print deprecation notices to the Rails logger.
   config.active_support.deprecation = :log
 
-  # Raises error for missing translations
+  # Raises error for missing translations.
   # config.action_view.raise_on_missing_translations = true
+
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.hosts << "asset-manager.dev.gov.uk"
```

Comment:
Unsure whether this should be `config.hosts.clear` or the following:

```
config.hosts = [
  "asset-manager.dev.gov.uk",
]
```

Trying to look through other repos for a pattern.

Reply:
As pointed in another comment, better to do with `+=` than `=`, so we don't override the addresses (like `127.0.0.1`) that are already on the array by default

Reply:
Yes, I changed my thinking 😄